packages/components/tests/dummy/app/templates/components/button-set.hbs

```diff
@@ -0,0 +1,77 @@
+{{page-title "ButtonSet Component"}}
+
+<h2 class="dummy-h2">ButtonSet</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">Provides consistent layout and spacing of buttons.</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
+  <p class="dummy-paragraph" id="component-api-button-set">
+    Here is the API for the component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-button-set">
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
+
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::ButtonSet>
+        <Hds::Button @text="Button 1" />
+        <Hds::Button @text="Button 2" />
+        <Hds::Button @text="Button 3" />
+      </Hds::ButtonSet>
+    '
+  />
+  {{! prettier-ignore-end }}
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="design-guidelines">
+    <a href="#design-guidelines" class="dummy-link-section">§</a>
+    Design guidelines
+  </h3>
+
+  <div class="dummy-design-guidelines">
```

Comment:
maybe we could add here the screenshot of the specific guideline related to the button set?
<img width="1440" alt="button-set" src="https://user-images.githubusercontent.com/686239/181187191-f7045499-39c9-4a35-9e65-a3420a6bd459.png">

Reply:
Thanks, I can do that

Reply:
@didoo Sorry, where should the screenshot image file go? I found a folder under "flight-website/public/assets/images" but that doesn't look like the right place...

Reply:
@KristinLBradley this is the folder: `packages/components/tests/dummy/public/assets/images/`
(the "scrappy" website lives under the "test/dummy" Ember app, is a special app used for testing, but we've used it for the website too).